realm-annotations/src/main/java/io/realm/annotations/PrimaryKey.java

```diff
@@ -25,10 +25,11 @@
  * The @PrimaryKey annotation will mark a field as a primary key inside Realm. Only one field in a
  * RealmObject class can have this annotation, and the field should uniquely identify the object.
  * Trying to insert an object with an existing primary key will result in an
- * {@link io.realm.exceptions.RealmPrimaryKeyConstraintException}.
- *
+ * {@link io.realm.exceptions.RealmPrimaryKeyConstraintException}. Primary key cannot be changed
```

Comment:
@Zhuinden That requires additional check and it doesn't seem to be useful ... user simply should avoid to call `setXXX` to set PK, right?